introduction.md

```diff
@@ -0,0 +1 @@
+Hi. I am José Cage. A web developer and product designer from Angola (African Country) and my favorite codeschool path is *Javascript*
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.